src/js/components/Select/doc.js

```diff
@@ -0,0 +1,60 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { a11yTitlePropType, backgroundPropType } from '../utils';
+
+export default (Select) => {
+  const DocumentedSelect = describe(Select)
+    .description(
+      'An select-like field with optional search capability.'
+    ).usage(
+      `import { Select } from 'grommet';
+      <Select/>`
+    );
+
+  DocumentedSelect.propTypes = {
+    a11yTitle: a11yTitlePropType,
+    background: backgroundPropType,
+    basis: PropTypes.string.description(
+      'Size of the options container inside the Select drop.'
+    ),
+    children: PropTypes.func.description(
+      'Function that will be called when each option is rendered.'
+    ),
+    name: PropTypes.string.description(
+      'Name of the select to be used inside a form context.'
+    ),
+    onChange: PropTypes.func.description(
+      'Function that will be called when the user selects an option.'
+    ),
+    onClose: PropTypes.func.description(
+      'Function that will be called when the Select drop closes.'
+    ),
+    onSearch: PropTypes.func.description(
+      `Function that will be called when the user types in the search input.
+      If this property is not provided, no search field will be rendered.`
+    ),
+    open: PropTypes.bool.description(
+      'Whether the Select drop should be open or not.'
+    ),
+    options: PropTypes.arrayOf(
+      PropTypes.oneOfType([PropTypes.string, PropTypes.element, PropTypes.object])
+    ).description(
+      `Options can be either a string or an object. If an object is used, use children callback
+      in order to render anything based on the current item.`
+    ).isRequired,
+    placeholder: PropTypes.string.description(
+      'Placeholder text to use when the input is empty.'
```

Comment:
'when the input is empty' -> 'when no value is provided'

Reply:
good